digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/IntegrationApi.scala

```diff
@@ -353,6 +369,31 @@ class IntegrationApi(val massTransitStopService: MassTransitStopService) extends
     }
   }
 
+  private def extractChangeType(since: DateTime, expired: Boolean, createdDateTime: Option[DateTime]) = {
+    if (expired) {
+      "Remove"
+    } else if (createdDateTime.exists(_.isAfter(since))) {
+      "Add"
+    } else {
+      "Modify"
+    }
+  }
+
+  get("/changes/:assetType") {
+    contentType = formats("json")
+     val since = DateTime.parse(params("since"))
```

Comment:
We should have something like params.getOrElse("since", halt(BadRequest("Missing mandatory 'since' parameter")))